### PR TITLE
OBPIH-5267 Create API architecture for React side for rest list pages

### PR DESCRIPTION
**Pull request by alannadolny**

### CODE REVIEW

File: src/js/api/services/StockListApi.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import {
+  STOCK_LIST_API,
+  STOCK_LIST_CLEAR,
+  STOCK_LIST_CLONE,
+  STOCK_LIST_DELETE,
+  STOCK_LIST_PUBLISH, STOCK_LIST_UNPUBLISH,
```
Comment (kchelstowski) [inline src/js/api/services/StockListApi.js:6]: we should keep stocklist as `STOCKLIST` as it is a proper name.
so:
```
STOCKLIST_API
```
etc.

**Review by kchelstowski — CHANGES_REQUESTED** [review]

**Comment by jmiranda** [comment]
@drodzewicz can you review this PR?

**Comment by jmiranda** [comment]
@drodzewicz nevermind, @kchelstowski says this one is ready to merge.

**Comment by awalkowiak** [comment]
This is good, but I think this will be a pain in the ass while rebasing grails 3 on top of this (but I'll manage it, hopefuly). Definitely this refactor should be done before starting the grails 3.

**Pull request merged**